docs/resources/Guild.md

```diff
@@ -945,10 +951,53 @@ Returns a PNG image widget for the guild. Requires no permissions or authenticat
+| ------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------ |
 | shield  | shield style widget with Discord icon and guild members online count                                                                                           | [Example](https://discord.com/api/guilds/81384788765712384/widget.png?style=shield)  |
 | banner1 | large image with guild icon, name and online count. "POWERED BY DISCORD" as the footer of the widget                                                           | [Example](https://discord.com/api/guilds/81384788765712384/widget.png?style=banner1) |
 | banner2 | smaller widget style with guild icon, name and online count. Split on the right with Discord logo                                                              | [Example](https://discord.com/api/guilds/81384788765712384/widget.png?style=banner2) |
 | banner3 | large image with guild icon, name and online count. In the footer, Discord logo on the left and "Chat Now" on the right                                        | [Example](https://discord.com/api/guilds/81384788765712384/widget.png?style=banner3) |
 | banner4 | large Discord logo at the top of the widget. Guild icon, name and online count in the middle portion of the widget and a "JOIN MY SERVER" button at the bottom | [Example](https://discord.com/api/guilds/81384788765712384/widget.png?style=banner4) |
+
+## Update Self Voice State % PATCH /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/voice-states/@me
+
+Updates the current user's voice state.
+
+###### JSON Params
+
+| Field                       | Type               | Description                                    |
+| --------------------------- | ------------------ | ---------------------------------------------- |
+| channel_id                  | snowflake          | the id of the channel the user is currently in |
+| suppress?                   | boolean            | toggles the user's suppress state              |
+| request_to_speak_timestamp? | ?ISO8601 timestamp | sets the user's request to speak               |
+
+###### Caveats
+
+There are currently several caveats for this endpoint:
+
+- `channel_id` must currently point to a stage channel.
+- current user must already have joined `channel_id`.
+- You must have the `MUTE_MEMBERS` permission to unsuppress yourself. You can always suppress yourself.
+- You must have the `REQUEST_TO_SPEAK` permission to request to speak. You can always clear your own request to speak.
+- You are able to set `request_to_speak_timestamp` to any present or future time.
```

Comment:
I'm not too happy with how I effectively listed a bunch of product behaviour here and below. Things that are mostly in flux too and probably improved in the future. Perhaps I should move this to another section/page? Not sure how we do things in other sections.

Reply:
Honestly, we don't have much consistency around this. If it's in flux, you can put a 

```
> warn
> This logic is still in testing and could be changed
```